src/makeCalculator.js

```diff
@@ -5,6 +5,53 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(num) {
+      result += num;
+
+      return this;
+    },
+    subtract(num) {
+      result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      result *= num;
+
+      return this;
+    },
+    divide(num) {
+      if (num !== 0) {
+        result /= num;
+      } else {
+        // eslint-disable-next-line no-console
+        console.error('Cannot divide by zero');
+      }
+    },
```

Comment:
The `divide` method should return `this` to maintain consistency with the other methods and allow method chaining. Add `return this;` after the division logic.